test/factories/books.js

```diff
@@ -0,0 +1,18 @@
+const factoryGirl = require('factory-girl'),
+  factory = factoryGirl.factory,
+  SequelizeAdapter = new factoryGirl.SequelizeAdapter(),
+  Book = require('../../app/models').book;
+
+factory.setAdapter(SequelizeAdapter);
+
+factory.define('Book', Book, {
+  name: factory.seq('Book.name', n => `book ${n}`)
+});
+
+exports.create = (options = {}) => factory.create('Book', options, {});
+
+exports.createMultiple = num => factory.createMany('Book', num);
```

Comment:
You can send some options to createMany too.

By the way, I don't know if you earn too much by defining all these functions for each factory.